salt/orch/kubernetes.sls

```diff
@@ -12,10 +12,11 @@
 {%- set additional_etcd_members = salt.caasp_etcd.get_additional_etcd_members(num_wanted=num_etcd_members,
                                                                               etcd_members=etcd_members) %}
 
-# Ensure the node is marked as bootstrapping
+# Ensure all the nodes are marked with a 'bootstrap_in_progress' flag
 set-bootstrap-in-progress-flag:
   salt.function:
-    - tgt: '*'
+    - tgt: 'roles:(ca|admin|kube-master|minion|etcd)'
```

Comment:
Typo here: `|minion|`

Reply:
I did the same typo I fixed some lines below... (faceplam)

Fixing.